corehq/apps/export/views.py

```diff
@@ -1673,6 +1675,16 @@ def _check_deid_permissions(self, export_instances):
                         "De-Identified export.")
                     )
 
+    def _check_export_size(self, export_instances, filters):
+        count = 0
+        for instance in export_instances:
+            count += get_export_size(instance, filters)
+        if count > MAX_EXPORTABLE_ROWS:
```

Comment:
nit: we may want to do this check in the for loop so we stop performing queries as soon as we're over the limit